test/optimus/scientific/ScientificTests.wdl

```diff
@@ -0,0 +1,24 @@
+
+# note that this is not a great test because it localizes the .bam file
+# however it is a good placeholder while we put together the scientific tests
+# themselves, as these files will be relatively small in the testing data.
+task TestBamRecordNumber {
+  File bam
+  Int expected_records
+  Int required_disk = ceil(size(bam, "G") * 2)
```

Comment:
you probably don't need to multiply by `2` here since its pretty much just the bam being pulled down